### DISCUSSION TITLE
Notebooks won't load on fresh install

### DISCUSSION THREAD

**Discussion by MoisesSierpe** [fonsp/Pluto.jl#2473] [Questions about using Pluto]
I was trying to make julia work in jupyter but the kernel wouldn't connect so I tried a bunch of things until I gave up, then went to use Pluto but no notebook would load. Unistalled-reinstalled julia and pluto but the problem persists, now I got no clue what to even google ... :(.
When I open pluto it loads but when I try to create or open a notebook it gives a short loading page and then stays on a blank screen, see image. I use firefox but tried chrome and it still doesn't work, the chrome installation is clean. There are no error messages in the julia shell, just some in the browser's console. Also, I don't know if the problem is recent since I haven't used Pluto in almost a year.

Does anyone have any idea of what's wrong?

![Screenshot 2023-02-09 163150](https://user-images.githubusercontent.com/15041630/217918540-a9617f8d-4d01-4e3a-9a63-0e100dff7e85.png)

**Comment by pankgeorg** [comment]
Can you disable any extension and try again? What version of Pluto are you using?
